variants/backend-base/config/environments/production.rb

```diff
@@ -8,14 +8,14 @@
   RUBY
 end
 
-gsub_file "config/environments/production.rb",
-          "# config.force_ssl = true",
-          <<~RUBY
-            ##
-            # `force_ssl` defaults to on. Set `force_ssl` to false if (and only if) RAILS_FORCE_SSL=false, otherwise set it to true.
-            #
-            config.force_ssl = ENV.fetch("RAILS_FORCE_SSL", "true").downcase != "false"
-          RUBY
+gsub_file! "config/environments/production.rb",
+           "config.force_ssl = true",
```

Comment:
note: Rails 7.1 now ships with this just enabled, but I think it's worth us keeping an env variable for toggling it just in case